js_modules/dagster-ui/packages/ui-core/src/graph/asyncGraphLayout.ts

```diff
@@ -139,7 +173,17 @@ export function useAssetLayout(graphData: GraphData) {
 
     async function runAsyncLayout() {
```

Comment:
This version also only runs if there are > 50 nodes based off the existing logic so we won't be caching __everything__, only expensive things.

Reply:
Doesn't need to change in this PR but I'd actually vote for folding these code paths and ALWAYS using the async version of this code. Because so few of our toys graphs have >50 assets, we almost never see the async code path in development which has led to at least one incident. I imagine that the time overhead of using the async path isn't that great... maybe there's a flickering loading indicator though?